pkg/ebpftracer/debug/debug_arm64_bpfel.go

```diff
@@ -0,0 +1,277 @@
+// Code generated by bpf2go; DO NOT EDIT.
+//go:build arm64
+
+package debug
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"io"
+
+	"github.com/cilium/ebpf"
+)
+
+type debugDebugSocketContext struct {
+	SockInfo debugSocketInfo
+	Netctx   debugNetTaskContext
+}
+
+type debugNetTaskContext struct{ Taskctx debugTaskContext }
+
+type debugProcessBpfMap struct {
+	MapId uint32
+	Name  [16]uint8
+}
+
+type debugSocketInfo struct {
+	Tuple  debugTupleT
+	Family uint16
+	Proto  uint8
+	State  uint8
+	_      [6]byte
+	Ino    uint64
+}
+
+type debugTaskContext struct {
+	StartTime       uint64
+	CgroupId        uint64
+	Pid             uint32
+	Tid             uint32
+	Ppid            uint32
+	HostPid         uint32
+	HostTid         uint32
+	HostPpid        uint32
+	NodeHostPid     uint32
+	Uid             uint32
+	MntId           uint32
+	PidId           uint32
+	Comm            [16]int8
```

Comment:
I wonder why does it generate [16]int8 instead of [16]byte? This doesn't allow to print as string without first converting to byte slice